hibike/devices/YogiBear/ui.cpp

```diff
@@ -0,0 +1,173 @@
+#include "Arduino.h" //to get digital write, etc.
+
+#include "ui.h"
+#include "motor.h"
+#include "pid.h"
+#include "encoder.h"
+#include "current_limit.h"
+//Code here that helps deal with the human-readable UI.
```

Comment:
Yes it is.  

We found it helpful during testing (esp when there were Times with hibike issues) to have a serial, almost command-line interface to the board.  We essentially kept the functionality which we wrote for arduino testing in.

I contend that it would be helpful to have more boards with similar functionality to that.  Anywho, that's the reason.